Create 2021-10-14-Subnero-at-Ocean-Business.md (#190)
subnero1/subnero1.github.io: _posts/2021-10-14-Subnero-at-Ocean-Business.md

---
layout: post

> title: Subnero attended Ocean Business 2021 in Southampton, UK

Reviewer: I would suggest to remove the place and country from the title as it would be a bit long and have it same as other similar posts. Maybe can change it to something like 'Subnero at Ocean Business 2021'.

Author: Agree, I'll amend accordingly.

banner : images/banner-pulse-standard.jpg
date:  2021-10-14
categories: wnc news
thumbnail: images/pulse-thumbnail-ocean-business-2021.jpg
---

Subnero attended its first in-person exhibition during [Ocean Business](https://noc-events.co.uk/ocean-business-2021){:target="_blank"}, in Southampton, UK.

> The event took place at the National Oceanography Centre from the 12th to the 14th of October 2021, after being postponed because of the pandemic. 

Reviewer 2: After being postponed from?

Author: added the original dates.

> It was so nice for the team to talk about our technology in person.

Reviewer 2: ```suggestion
It was nice for the team to meet other conference attendees and talk about our technology in person.
```

> It was so nice to meet for the team to talk about our technology in person.

Reviewer: ```suggestion
It was so nice to meet the team to talk about our technology in person.
```

Author: There is a typo here "meet" should be there. I'll correct the entire sentence.

We loved being part of the event.

> We look forward to attending the next in person exhibition [Oceanology International 2022](https://www.oceanologyinternational.com/london/en-gb.html), in London, UK, in March 2022.

Reviewer: ```suggestion
We look forward to attending the next in person exhibition [Oceanology International 2022](https://www.oceanologyinternational.com/london/en-gb.html){:target="_blank"}, in London, UK, in March 2022.
```
